### PR TITLE
Add args parser

### PR DESCRIPTION
**Pull request by andlft**

### CODE REVIEW

**Comment by Qmpzlawasd** [comment]
pare scris de lefter

**Comment by Qmpzlawasd** [comment]
lgtm

**Comment by andlft** [comment]
:notlikethis:

**Comment by AdelaCorbeanu** [comment]
sal

**Comment by andlft** [comment]
dark?

**Review by Qmpzlawasd — APPROVED** [review]

**Review by AdelaCorbeanu — APPROVED** [review]

File: arg_parse.c [hunk 1]
```diff
@@ -0,0 +1,110 @@
+#include <stdio.h>   
+#include <stdlib.h>    
+#include <getopt.h>
+#include <string.h>
+#define EXIT_SUCCESS 0
+#define EXIT_FAILURE 1
```
Comment (tezkerek) [inline arg_parse.c]: Already defined in stdlib.h

**Review by tezkerek — CHANGES_REQUESTED** [review]

**Review by tezkerek — APPROVED** [review]
LGTM

**Pull request merged**